package/kernel/mac80211/patches/build/211-define-backport-kernel-code.patch

```diff
@@ -0,0 +1,71 @@
+--- /dev/null
++++ b/scripts/backport-version-code.pl
+@@ -0,0 +1,19 @@
++#!/usr/bin/env perl
++# SPDX-License-Identifier: GPL-2.0
++
++use strict;
++
++while (<>) {
++  my $line = $_;
++  $line =~ /.*([[:digit:]])+\.([[:digit:]]+)\.([[:digit:]]+).*/;
++  if ($1 and $2 >= 0 and $3 >= 0) {
```

Comment:
Need to fix this for `-rc` kernel versions, where there is not sublevel in the git tag:
```sh
cat backports-5.8-rc2-1/versions 
BACKPORTS_VERSION="v5.8-rc2-1-0-g7a69429b"
BACKPORTED_KERNEL_VERSION="v5.8-rc2-0-g48778464bb7d"
BACKPORTED_KERNEL_NAME="Linux"
```